filesystem/src/main/java/com/nytimes/android/external/fs/BarCodePathResolver.java

```diff
@@ -7,7 +7,7 @@
 class BarCodePathResolver implements PathResolver<BarCode> {
     @Nonnull
     @Override
-    public String resolve(@Nonnull BarCode key) {
-        return key.toString();
+    public String resolve(@Nonnull BarCode barCode) {
+        return barCode.getType()+"/"+barCode.getKey();
```

Comment:
kept going back and forth on whether `toString` should have the "/" since that is only an implementation detail of the Reader.